File: app/src/main/java/com/pitchai/flickrrecentphotobrowser/presentation/view/GridPhotoListView.java
package com.pitchai.flickrrecentphotobrowser.presentation.view;

import com.pitchai.flickrrecentphotobrowser.data.photoinfo.Photo;

import java.util.List;

/**
 * Created by pitchairajamani on 2/27/17.
 */

public interface GridPhotoListView extends LoadDataView{
    void renderPhotoList(List<Photo> photoList, int currentPageNumber);
}

File: app/src/main/java/com/pitchai/flickrrecentphotobrowser/presentation/view/GridPhotoListView.kt
package com.pitchai.flickrrecentphotobrowser.presentation.view

import com.pitchai.flickrrecentphotobrowser.data.photoinfo.Photo

/**
 * Created by pitchairajamani on 2/27/17.
 */
interface GridPhotoListView : LoadDataView {
    fun renderPhotoList(photoList: List<Photo?>?, currentPageNumber: Int)
}